librz/bin/p/bin_pe.inc

```diff
@@ -26,6 +26,21 @@ static bool load_buffer(RzBinFile *bf, RzBinObject *obj, RzBuffer *buf, Sdb *sdb
 	if (res) {
 		sdb_ns_set(sdb, "info", res->kv);
 		obj->bin_obj = res;
+
+		// Hint on possible bogus files if no loading addr was given
+		if (obj->opts.baseaddr == UT64_MAX) {
+			ut64 ImageBase = PE_(rz_bin_pe_get_image_base)(res);
+			if (ImageBase < 0x10000) {
+				RZ_LOG_INFO("%s",
+					"ImageBase below 0x10000 is invalid on Windows.\n"
+					"Preferred addresses are:\n"
+					"0x10000000 DLLs\n"
+					"0x00010000 Windows CE EXEs\n"
+					"0x00400000 Windows NT, 2000, XP, 95, 98, and Me\n"
+					"Use the -B option to set base address.\n"
+				);
+			}
+		}
```

Comment:
I would avoid this TBH, it is very system dependent and one should know what to do with its own files to reverse. Moreover, this is just windows specific and if I'm analyzing a EFI app I don't want to see this.

Reply:
I don't insist in keeping that

Reply:
At least this is the right location for some auto magic.